web-registry/src/components/templates/ProjectDetails/ProjectDetails.tsx

```diff
@@ -269,21 +275,31 @@ function ProjectDetails(): JSX.Element {
         projectMetadata={projectMetadata}
         projectPageMetadata={offChainProjectMetadata}
         sanityCreditClassData={sanityCreditClassData}
+        geojson={geojson}
+        isGISFile={isGISFile}
+        onChainProjectId={onChainProjectId}
+        projectDeveloper={projectDeveloper}
+        landOwner={landOwner}
+        landSteward={landSteward}
+        loading={loadingDb || loadingAnchoredMetadata}
         soldOutProjectsIds={soldOutProjectsIds}
         projectWithOrderData={projectsWithOrderData[0]}
         batchData={{
           batches: batchesWithSupply,
           totals: batchesTotal,
         }}
-        projectDeveloper={projectDeveloper}
-        landOwner={landOwner}
-        landSteward={landSteward}
+      />
+
+      {hasProjectPhotos && <Gallery photos={projectPhotos} />}
+
+      <ProjectMiddleSection
```

Comment:
actually not sure if we should call this "middle" section, since the batches table will eventually go down the project page within the credit issuance tab: https://www.figma.com/file/seUiQM2SPt78KJiDL0Ea8m/Basic-Ecocredits-Ledger-View-%2F-Marketplace?node-id=6668-173353&t=SKOIJp1WEJK10YR7-0

Reply:
Ok, will rename it `CreditBatchesSection` since it only holds this block now.

Reply:
sounds good, could you move it down too?